.travis.yml

```diff
@@ -28,8 +26,11 @@ before_install:
  - sudo dpkg -i google-chrome*.deb
  - rm -fr /home/travis/.m2/repository/webdriver/
 
-after_success:
-- ./gradlew jacocoTestReport coveralls
-- bash <(curl -s https://codecov.io/bash)
+script:
+  - ./gradlew check chrome htmlunit --no-daemon --console=plain
+  - ./gradlew sonarqube --no-daemon --console=plain -Dsonar.organization=${SONAR_ORG} -Dsonar.host.url=${SONAR_HOST} -Dsonar.login=${SONAR_LOGIN} -Dsonar.branch.name=${TRAVIS_BRANCH}
 
-after_script: "./.upload_reports.sh"
```

Comment:
is it ok? does sonar show failed tests stacktrace and etc?

Reply:
Sonar isn't.
Failed tests stacktraces will on travis side.

Reply:
`"./.upload_reports.sh"` uses to upload test artifacts (in fact /build/reports/ folder) for test failure investigations. Please come this back